.github/workflows/engine_helper_functions_test.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       run: cmake --build ${{env.ENGINE_DIR}}/build --target main -j$(nproc)
 
     - name: Setup environment
-      run: python3 ${{env.ENGINE_DIR}}/test/setupEnvironment.py -e /tmp/actions
+      run: sudo python3 ${{env.ENGINE_DIR}}/test/setupEnvironment.py -e /tmp/actions
```

Comment:
If you run the script with sudo, then the created folders will be owned by root, so the scripts should be run as sudo as well to avoid getting permissions denied.


i.e.
```bash
ubuntu@5a6599db6cc7:/workspaces/repos$ mkdir /tmp/noSudo
ubuntu@5a6599db6cc7:/workspaces/repos$ sudo mkdir /tmp/sudo
[sudo] password for ubuntu: 
ubuntu@5a6599db6cc7:/workspaces/repos$ ls -la /tmp/noSudo
total 0
drwxrwxr-x  2 ubuntu ubuntu   40 Oct 14 23:40 .
drwxrwxrwt 19 root   root   3040 Oct 14 23:40 ..
ubuntu@5a6599db6cc7:/workspaces/repos$ ls -la /tmp/sudo
total 0
drwxr-xr-x  2 root root   40 Oct 14 23:40 .
```